Py/DICOMTID1500Plugin.py

```diff
@@ -127,19 +127,22 @@ def load(self, loadable):
         if hasattr(segLoadable, "referencedSeriesUID"):
           # determine if RWVM needs to be applied to the series referenced from SEG
           loadRWVM = False
-          if len(loadable.ReferencedRWVMSeriesInstanceUIDs)>0:
+          rwvmUIDs = loadable.ReferencedRWVMSeriesInstanceUIDs
+          if len(rwvmUIDs)>0 and len(slicer.dicomDatabase.filesForSeries(rwvmUIDs[0]))>0:
```

Comment:
I wouldn't run filesForSeries that often since there will always be executed another query on the dicomDatabase [1]. I would reuse the value after having run it once

[1] https://github.com/commontk/CTK/blob/04607d0ea7fbab104ede38c7c2acc7b5dd1fdfa0/Libs/DICOM/Core/ctkDICOMDatabase.cpp#L786-L796